simulation/model.py

```diff
@@ -34,6 +36,9 @@ def __init__(
         self.heating_systems = set(HeatingSystem)
         self.household_num_lookahead_years = household_num_lookahead_years
         self.heating_system_hassle_factor = heating_system_hassle_factor
+        self.intervention = (
+            InterventionType[intervention.upper()] if intervention else None
+        )
```

Comment:
Ok for now, if we only have one intervention type (as we do), but perhaps this becomes a list of potential `InterventionTypes` in the future. Then, in the `Household` agent you could check for `InterventionType.RHI in model.interventions`.

It might be that we change this approach completely for more complicated interventions, and I think its fine for now.

Reply:
Yeah, I think b/c of the slight uncertainty about ever needing to 'layer' interventions, I'm proposing we keep it simple -- just account for the known use case.